src/v2/Apps/Order/Components/__tests__/PriceOptions.jest.tsx

```diff
@@ -124,15 +124,13 @@ describe("PriceOptions", () => {
         )
       )
       fireEvent.click(radios[3])
-      expect(trackEvent).toHaveBeenLastCalledWith(
+      expect(trackEvent).toHaveBeenCalledWith(
```

Comment:
I switched this since the click event, and the "viewed offer too low" event are triggered simultaneously.